scalikejdbc-core/src/main/scala/scalikejdbc/DBSession.scala

```diff
@@ -190,6 +196,28 @@ trait DBSession extends LogSupport with LoanPattern {
   def tags: Seq[String] = this._tags
 
   /**
+   * Set queryTimeout to this session.
+   * @param seconds query timeout seconds
+   * @return this
+   */
+  def queryTimeout(seconds: Int): DBSession = {
```

Comment:
I see.
Should I change `def fetchSize(fetchSize: Int)`, `def fetchSize(fetchSize: Option[Int])`, `def tags(tags: String*)` in the same way?